src/components/menu/AccessSubmenu/index.js

```diff
@@ -1,27 +1,27 @@
 import React from "react";
+import { useSelector, useDispatch } from "react-redux";
 import List from "@material-ui/core/List";
 import ListItem from "@material-ui/core/ListItem";
 import ListItemText from "@material-ui/core/ListItemText";
 import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction";
 import Switch from "@material-ui/core/Switch";
-import { connect } from "react-redux";
 import { listenToAccessToggle } from "../../../redux/actions";
 
-const ACCESS_PROPERTIES = [
-    "housing",
-    "parks",
-    "nightlife",
-    "employment",
-    "groceries",
-    "restaurants",
-    "education"
-];
+function AccessSubmenu() {
+    const { accessToggle, cityioData } = useSelector(state => ({
```

Comment:
does useSelector get global state from redux?Since AccessSubmenu is a child of TogglesMenu, wouldn't it be possible to get cityIO state  as props?